app/assets/stylesheets/course/enrol_requests.css.scss

```diff
@@ -0,0 +1,46 @@
+#enrol_request_form {
+  min-width: 190px;
+  width: 190px;
+
+  a {
+    margin: 5px 0px 5px 0px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `5px 0px` instead of `5px 0px 5px 0px`<br>`0px` should be written without units as `0`